autogpt_platform/backend/backend/data/db.py

```diff
@@ -62,10 +62,10 @@ async def connect():
 
     # Connection acquired from a pool like Supabase somehow still possibly allows
     # the db client obtains a connection but still reject query connection afterward.
-    try:
-        await prisma.execute_raw("SELECT 1")
-    except Exception as e:
-        raise ConnectionError("Failed to connect to Prisma.") from e
+    # try:
```

Comment:
@majdyz I don't think we need this anymore, is this all fine to remove or does it do additional work other than checking the connection